roadmap.md

```diff
@@ -5,44 +5,44 @@ The CloudEvents Roadmap.
 *Setup*
 
 * Establish governance, contributing guidelines and initial stakeholders.
-* Define design goals for CloudEvents V.1
-* Describe the scope of the specification
+* Define design goals for CloudEvents V.1.
+* Describe the scope of the specification.
 * Draft educational materials that provide context for reading the spec.
 
 *0.1*
 
-* Draft specification that WG members agree *could* provide interoperability
-* Document at least 3 sample events that conform to the specification
-* Github repo is organized to be approachable to a engineers who might want to implement the spec
+* Draft specification that WG members agree *could* provide interoperability.
+* Document at least 3 sample events that conform to the specification.
+* Github repo is organized to be approachable to a engineers who might want to
+implement the spec.
 * Additional materials:
-  * use-cases
-  * placeholder for protocol specification, folder and brief starter doc for HTTP
+  * Use-cases.
+  * placeholder for protocol specification, folder and brief starter doc for
```

Comment:
Capitalize "Placeholder" while you are making this consistent?

Reply:
Done.